src/index.js

```diff
@@ -0,0 +1,152 @@
+'use strict';
+
+const tempDisplay = document.getElementById('temp-display');
+const landscapeImg = document.getElementById('landscape-img');
+const cityName = document.getElementById('city-name-display');
+const cityInputBox = document.getElementById('city-name');
+
+const state = {
+  temp: 80,
+  city: cityName.textContent,
+  color: 'rgb(248, 163, 163)',
+};
+
+// Set default values
+tempDisplay.textContent = `${state.temp}°`;
+tempDisplay.style.color = state.color;
+
+const updateTempDisplay = (x) => {
+  state.temp += x;
+  tempDisplay.textContent = `${state.temp}°`;
+
+  if (state.temp > 79) {
+    tempDisplay.style.color = 'rgb(248, 163, 163)';
+  }
+  if (state.temp < 80) {
+    tempDisplay.style.color = 'orange';
+    landscapeImg.src = '/images/sunny.png';
+    landscapeImg.alt = 'black line drawings of sun, with circle in center';
+  }
+  if (state.temp < 70) {
+    tempDisplay.style.color = 'yellow';
+    landscapeImg.src = '/images/cloudy.png';
+    landscapeImg.alt = 'black line drawing of clouds in front of a sun';
+  }
+  if (state.temp < 60) {
+    tempDisplay.style.color = '#2B835E';
```

Comment:
In production code we'd likely want to include comments about what the hex colors are, since many folks cannot sight-read hex values.